architecture/wm_gcp/Readme.md

```diff
@@ -0,0 +1,24 @@
+<!---
+Copyright (C) 2015-2021, Wazuh Inc.
+Created by Wazuh, Inc. <info@wazuh.com>.
+This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+-->
+
+# Wazuh module: Google Cloud
+## Index
+1. [Purpose](#purpose)
+2. [Sequence diagram](#sequence-diagram)
+
+## Purpose
+The Google Cloud Module is in charge of requests events to the Google Cloud API and deals with them. The module get the events from a Google Cloud subscription that is specified in the configuration file.

```

Comment:
- `requests` -> `requesting`
- `deals with them` -> `forward them to the manager.`